src/component/SortedLists.js

```diff
@@ -0,0 +1,177 @@
+        filter={[
+          ['numberOfDays', '>=', 7],
+          ['numberOfDays', '<=', 30],
+        ]}
+        render={({ isLoading, data }) => {
+          if (isLoading) {
+            return <div>Still Loading...</div>;
+          } else {
+            return (
+              <div id="kindOfSoonItems">
+                <h2 className="itemsLabel">Kind of Soon Items</h2>
+                <ul>
+                  {data.map(item => (
+                    <li
+                      id={item.id}
+                      key={item.id}
+                      className={calculateIfPurchased(item)}
+                    >
+                      <div
+                        className={item.name}
+                        onClick={handlePurchase}
+                        id={item.id}
+                        aria-label="Kind of soon Item"
+                        aria-required="true"
+                      >
+                        {item.name}
+                      </div>
+                    </li>
+                  ))}
+                </ul>
+              </div>
+            );
+          }
+        }}
+      />
+      {/* items that need to buy not soon (greater than 30 days)  */}
+      <FirestoreCollection
+        path={concatPath}
+        sort="numberOfDays:asc"
+        filter={[
+          ['numberOfDays', '>', 30],
+          ['numberOfDays', '<', 60],
+        ]}
+        render={({ isLoading, data }) => {
+          if (isLoading) {
+            return <div>Still Loading...</div>;
+          } else {
+            return (
+              <div id="notSoonItems">
+                <h2 className="itemsLabel">Not soon Items</h2>
+                <ul>
+                  {data.map(item => (
+                    <li
+                      id={item.id}
+                      key={item.id}
+                      className={calculateIfPurchased(item)}
+                    >
+                      <div
+                        className={item.name}
+                        onClick={handlePurchase}
+                        id={item.id}
+                        aria-label="Not so soon Item"
+                        aria-required="true"
+                      >
+                        {item.name}
+                      </div>
+                    </li>
+                  ))}
+                </ul>
+              </div>
+            );
+          }
+        }}
+      />
+      {/* items that are inactive (inactive meaning only one purchase or purchase is out date) */}
+      <FirestoreCollection
+        path={concatPath}
+        sort="numberOfDays:asc"
+        render={({ isLoading, data }) => {
+          if (isLoading) {
+            return <div>Still Loading...</div>;
+          } else {
+            const filteredItem = data.filter(item => {
+              let doubleEstimate = item.numberOfDays * 2;
+              let lastPurchaseDate = item.dateOfPurchase.toDate();
+              let doublePurchaseEstimate = dayjs(lastPurchaseDate)
```

Comment:
You can use `const` here instead since you're not redefining the value of these 3 values anywhere else